arrows/CMakeLists.txt

```diff
@@ -102,3 +102,6 @@ if( KWIVER_ENABLE_VTK)
 endif()
 
 add_subdirectory( serialize )
+
+# Register the colmap arrow with CMake.
+add_subdirectory( colmap )
```

Comment:
This need to be wrapped in ```if```, ```endif``` with KWIVER_ENABLE_COLMAP. See what was done for Ceres:

https://github.com/Kitware/kwiver/blob/master/arrows/CMakeLists.txt#L17-L20

Reply:
This still needs to be wrapped.

```
if ( KWIVER_ENABLE_COLMAP )
  add_subdirectory( colmap )
endif()
```